src/app/views/river-index/components/nwi-map.vue

```diff
@@ -460,6 +487,21 @@ export default {
       })
     },
     initMap () {
+      if (isAmplifyDeploy && this.tileserverToken === null) {
+        const url = 'https://tileserver.americanwhitewater.org/token/';
+        const that = this;
+        fetch(url)
```

Comment:
obviously this is a pretty simple response so it's probably fine to use `fetch` but we do bundle an http library called [axios](https://github.com/axios/axios) that you could use here

Reply:
to add to this, we don't use `axios` natively in most cases, we've wrapped it in a module includable with `import http from "@/app/http"`